app/backend/resources/schema.edn

```diff
@@ -1817,7 +1817,86 @@
+
+          {:db/ident :thk.contract/procurement-number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Procurement no /Hanke nr - (if starts with number then converted into link to State procurement registry as https://riigihanked.riik.ee/rhr-web/#/procurement/<proc_No>), if starts with other symbol then leave it as a number)"}
+
+          ;; Contract statuses
+          {:db/ident :thk.contract.status/contract-signed
+           :enum/attribute :thk.contract/status}
+          {:db/ident :thk.contract.status/in-progress
+           :enum/attribute :thk.contract/status}
+          {:db/ident :thk.contract.status/warranty-period
+           :enum/attribute :thk.contract/status}
+          {:db/ident :thk.contract.status/completed
+           :enum/attribute :thk.contract.response/status}
+
+          {:db/ident :thk.contract/status
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "Enum reference to contract status"}
+
+          {:db/ident :thk.contract/region
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "The region of the object is displayed"}
+
+          ;; Contract types
+          {:db/ident :thk.contract.type/construction-works
+           :enum/attribute :thk.contract/type}
+          {:db/ident :thk.contract.type/services
+           :enum/attribute :thk.contract/type}
+          {:db/ident :thk.contract.type/things
+           :enum/attribute :thk.contract/type}
+
+          {:db/ident :thk.contract/type
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "Comes from THK, not editable (activity_procurementType, activity_procurementTypeFK"}
+
+          {:db/ident :thk.contract/signed-at
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "Then contract signed, added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/deadline
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "The first deadline, added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/authority-contact
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/partner-name
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Added manually in the detail view of contract. Lead partner only."}
+
+          {:db/ident :thk.contract/cost
+           :db/valueType :db.type/bigdec
+           :db/cardinality :db.cardinality/one
+           :db/doc "Added manually, the same as in the detail view: capital amount in euro"}
+
+          {:db/ident :thk.contract/project
```

Comment:
Change to `thk.contract/targets` if we actually need to have contracts targeting activities and projects
and change to `db.cardinality/many`

Reply:
Targets can based on the current understanding be either activities or tasks.